WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -725,6 +730,29 @@ private extension EditableOrderViewModel {
             .assign(to: &$shouldShowNonEditableIndicators)
     }
 
+    /// Binds the order state to the `multipleLineMessage` property.
+    ///
+    func configureMultipleLinesMessage() {
+        Publishers.CombineLatest(orderSynchronizer.orderPublisher, Just(flow))
+            .map { order, flow -> String? in
+                switch (flow, order.shippingLines.count, order.fees.count) {
+                case (.creation, _, _):
+                    return nil
+                case (.editing, 2...Int.max, 0...1): // Multiple shipping lines
+                    return Localization.multipleShippingLines
+                case (.editing, 0...1, 2...Int.max): // Multiple fee lines
+                    return Localization.multipleFeeLines
+                case (.editing, 2...Int.max, 2...Int.max): // Multiple shipping & fee lines
+                    return Localization.multipleFeesAndShippingLines
+                case (.editing, _, _): // Negative count - Should not happen!
```

Comment:
The comment here mentions negative fees but it also covers the case of 0 or 1 shipping and fee line, right?

Reply:
Agree. I don't think we should mention negative values, since it's coming from `Array.count` and expected to be non-negative.
✅ d16204d